style.css

```diff
@@ -131,8 +131,92 @@ a {
   text-decoration: underline;
 }
 
+/* wrapper for side navbar and component display section  */
+.wrapper {
+  width: 90%;
+  height: 100vh;
+  margin: 0 auto;
+  
```

Comment:
You could avoid an empty line here also and at multiple occurrences in this same file.

Reply:
okay.